os_migrate/roles/conversion_host/tasks/delete.yml

```diff
@@ -11,6 +11,13 @@
     ca_cert: "{{ os_migrate_conversion_ca_cert|default(omit) }}"
     client_cert: "{{ os_migrate_conversion_client_cert|default(omit) }}"
     client_key: "{{ os_migrate_conversion_client_key|default(omit) }}"
+  loop:
```

Comment:
From the playbook we get some undefined errors, but from here https://gitlab.cee.redhat.com/os-migrate/bot/-/jobs/4400905#L1869 it looks like it works correctly.